saleor/graphql/webhook/schema.py

```diff
@@ -21,6 +22,19 @@ class WebhookQueries(graphene.ObjectType):
         filter=WebhookFilterInput(description="Filtering options for webhooks."),
     )
 
+    webhook_sample_payload = graphene.Field(
```

Comment:
Missing description for this field. I would explain that sample payloads are used by some integrations and it returns real data.

Reply:
🚓 👮